src/cpp/src/whisper_pipeline_static.cpp

```diff
@@ -640,14 +665,22 @@ WhisperDecodedResults WhisperPipeline::StaticWhisperPipeline::generate(
             m_models.decoder = m_decoder_cache.get_model(init_ids.size());
         }
 
-        auto [cancelled, chunk_output_tokens] = full_decode(hidden_state_tensor,
-                                                            config,
-                                                            m_models,
-                                                            init_ids,
-                                                            max_new_tokens - output_tokens.size(),
-                                                            return_timestamps,
-                                                            raw_metrics,
-                                                            streamer_ptr);
+        std::vector<int64_t> chunk_init_tokens;
+        chunk_init_tokens.insert(chunk_init_tokens.end(), init_ids.begin(), init_ids.end());
+
+        SequenceGroup::Ptr sequence_group = std::make_shared<SequenceGroup>(0, chunk_init_tokens, config, 1);
```

Comment:
`chunk_init_tokens` looks redundant. Can't we pass `init_ids` direclty?

Reply:
They have different types, std::vector<int64_t> and std::vector<int32_t> accordingly.

Reply:
Is it possible to change `init_ids` to `std::vector<int64_t>`? I see `static_cast`s in `prepare_init_ids` are they required?

Reply:
Looks like there's no reason against it. Changed to std::vector<int64_t>.